k2/csrc/arcsort.cc

```diff
@@ -11,7 +11,7 @@
 #include <utility>
 #include <vector>
 
-#include "glog/logging.h"
+#include "k2/util/Logging.h"
```

Comment:
please follow the code style and sort the headers
in alphabetic order.

Reply:
All filenames should be in `lowercase`.

Reply:
It's what may differ with logging.* in glog. But on case-insensitive platforms (windows), it's useless. Thus I changed to lowercase.

Changed to the alphabetic order.